auth0/src/test/java/com/auth0/android/request/DefaultClientTest.kt

```diff
@@ -163,20 +222,21 @@ public class DefaultClientTest {
             HttpMethod.DELETE -> assertThat(request.method, equalTo("DELETE"))
         }
         assertThat(requestUri.path, equalTo(URL_PATH))
+
         if (method == HttpMethod.GET) {
             assertThat(requestUri.getQueryParameter("customer"), equalTo("john-doe"))
             assertThat(request.bodyFromJson(), anEmptyMap())
         } else {
             assertThat(requestUri.query, nullValue())
             assertThat(request.bodyFromJson(), hasEntry("customer", "john-doe"))
         }
-        assertThat(
-            request.headers.toMultimap(),
-            hasEntry(
-                equalTo("a-header"),
-                hasItem("b-value")
-            )
-        )
+
+        val requestHeaders = request.headers.toMultimap()
+        headers.forEach { (k, v) ->
+            assertThat(requestHeaders, hasEntry(
+                equalTo(k), hasItem(v)
```

Comment:
Given that the multimap is `String, List<String>` I think that `hasEntry` is not enough to check that we're actually preferring request headers over global ones., as they could both exist under the same key. We probably should check for `size == 1` as well.

Reply:
Updated to check the size.